templates/zerver/help/saml-authentication.md

```diff
@@ -57,6 +57,57 @@ Zulip with various common SAML identity providers.
      * Optionally you can also send us an icon that should be shown on the button.
 1. We will take care of the server-side setup and let you know as soon as it's ready.
 
+## Configure SAML with Keycloak
+
+1. Make sure you have created your organization. We'll assume its URL is
+   `https://<subdomain>.zulipchat.com` in the instructions below.
+2. Make sure your Keycloak server is up and running. We assume the URL to your keycloak is
+   `https://keycloak.example.com` and your realm is `master` (which is the default realm)
+
+
+1. Export the **public certificate of your IdP**: \
+   Navigate to the realm settings, click on the tab `Keys` and then on the button `Certificate`. Copy the content and
+   save it as `{idp_name}.crt`:
+   ```
+   -----BEGIN CERTIFICATE-----
+   {Paste the content here}
+   -----END CERTIFICATE-----
+   ```
+2. Create a new client:
+    * Client-ID: `https://<subdomain>.zulipchat.com`
+    * Client Protocol: saml
+    * Client SAML Endpoint: leave this field empty
+3. Change some configurations of your keycloak client: \
+   Switch to the configurations tab `Settings`:
+    - Valid Redirect URIs: `https://<subdomain>.zulipchat.com/*`
+    - Base URL: `https://<subdomain>.zulipchat.com/complete/saml/`
```

Comment:
Shouldn't these be on the `auth.zulipchat.com` subdomain, given this is a multi-org server, so it uses `SOCIAL_AUTH_SUBDOMAIN`?

Reply:
You are right, I forgot about that. How is this handled for multi-org servers? Where should keycloak redirect the user to after a successful login?

Reply:
It'll just be `https://auth.zulipchat.com/complete/saml/`